src/reuse/modules/mobile/userInteraction.ts

```diff
@@ -14,9 +14,11 @@ export class UserInteraction {
    * @function tap
    * @memberof mobile.userInteraction
    * @description Tap's on the mobile element.
-   * @param {Element | string} element - The element or CSS selector describing the element.
-   * @param {Number} [timeout=30000] - The timeout to wait (ms).
-   * @example const elem = await mobile.userInteraction.tap(elem);
+   * @param {Element | string} element - The element (e.g., accessibility ID, XPath, CSS,) selectors describing the element.
+   * @param {Number} [timeout=30000] - The timeout to wait(ms) (default: 30000)
```

Comment:
I guess there is no need to specify the default value, since it will already be shown by JSDoc
```suggestion
   * @param {Number} [timeout=30000] - The timeout to wait(ms)
```